src/test_helpers/component.cljs

```diff
@@ -71,3 +72,7 @@
 (defn is-null
   [element]
   (.toBeNull (js/expect element)))
+
+(defn was-called
```

Comment:
What do you think about this helper function added?

Reply:
I don't have any particular opinion @ulisesmac. I think the name you chose is consistent with the others, like `is-null`.